src/Billing/Controllers/StripeController.cs

```diff
@@ -3,7 +3,6 @@
 using Bit.Billing.Services;
 using Bit.Core;
 using Bit.Core.AdminConsole.Entities;
-using Bit.Core.AdminConsole.Enums.Provider;
```

Comment:
I logged a separate PR for this lint failure stuff [here](https://github.com/bitwarden/server/pull/4084)